Sample_v3/Samples/CombineSimpleChat/CombineSimpleChatViewController.swift

```diff
@@ -257,7 +257,7 @@ final class CombineSimpleChatViewController: UITableViewController, UITextViewDe
         let message = channelController.messages[indexPath.row]
         
         let messageController = channelController.client.messageController(
-            cid: channelController.channelQuery.cid,
+            cid: channelController.cid!,
             messageId: message.id
         )
```

Comment:
Please, let's use `if let` in the sample code ;-) If it's not possible for `cid` to be `nil` like in this case (?), we can make it a `fatalError`.